docs/model/azure.rst

```diff
@@ -0,0 +1,5 @@
+Push items: VHD
```

Comment:
A change will be needed to docs/index.rst as well in order to make these pages reachable.

Reply:
Fixed in `index.rst`, PTAL